governance.md

```diff
@@ -0,0 +1,325 @@
+-   Financial interests, such as investments, employment or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+
+All members of the Core Team, BDFL included, shall disclose to the rest of the
+Core Team any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Core Team discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+### Private communications of the Core Team
+
+Unless specifically required, all Core Team discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Core Team will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Core Team will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+### Subcommittees
+
+The Core Team can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Like the Core Team as a whole, subcommittees
+should conduct their business in an open and public manner unless privacy is
+specifically called for. Private subcommittee communications should happen on
+the main private mailing list of the Core Team unless specifically called for.
+
+Question: if the BDFL is not on a subcommittee, do they still have override
+authority?
+
+Suggestion: they do, but they should appoint a delegate who plays that role
+most of the time, and explicit BDFL intervention is sought only if the
+committee disagrees with that delegate’s decision and no resolution is possible
+within the team. This is different from a BDFL delegate for a specific decision
+(or a recusal situation), where the BDFL is literally giving up his/her
+authority to someone else in full. It’s more like what Linus Torvalds uses with his
+“lieutenants” model.
+
+### NumFOCUS Subcommittee
+
+The Core Team will maintain one narrowly focused subcommittee to manage its
+interactions with NumFOCUS.
+
+- The NumFOCUS Subcommittee is comprised of 5 persons who manage project
+  funding that comes through NumFOCUS. It is expected that these funds will be
+  spent in a manner that is consistent with the non-profit mission of NumFOCUS
+  and the direction of the Project as determined by the full Core Team.
+- This Subcommittee shall NOT make decisions about the direction, scope or
+  technical direction of the Project.
+- This Subcommittee will have 5 members. No more than 2 Subcommitee Members can
+  report to one person through employment or contracting work (including the
```

Comment:
1. I don't think that should matter, since the goal here is to prevent any particular financial entity from effectively controlling how money donated to pandas development is spent.

2. Probably best to clarify that "reports to" can include indirect reporting lines (e.g. to a senior manager of VP/C-level executive).